Client/src/components/contanier/staff/Student/ViewStudent/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "./index.css";
+import Table from "../../../../common/Table";
+import Button from "../../../../common/Button";
+import Input from "../../../../common/Input";
+
+export default class ViewStudent extends Component {
+  render() {
+    const headings = [
+      "Student Name",
+      "Student Number",
+      "Level ",
+      "Mobile Number"
+    ];
+    const rows = [
```

Comment:
@naremanhilles I think I said this already in another PR but this is a good way to stay productive when the endpoints are still being developed.
You can even put it into the component's state, since that is where the data will eventually live, and then have render() use this.state.rows - that will save you a step later when the endpoint is ready to be fetched.

Reply:
thx i fexed it :)

Reply:
@naremanhilles 👏thanks for doing that!